src/olympia/migrations/1017-add-version-to-reviewerscore.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE `reviewer_scores`
+	ADD COLUMN `version_id` INTEGER UNSIGNED REFERENCES `versions` (`id`);
```

Comment:
typical comment we add is:
`-- Note: if the migration fails for you locally, remove the 'UNSIGNED' next to version_id below.`

Reply:
Done in 57b2f7f15f6e8303163b49622cb3a64e551497a1.